tron/bin/check_tron_jobs.py

```diff
@@ -286,14 +286,17 @@ def guess_realert_every(job):
             return -1
         job_runs = job.get('runs', [])
         job_runs_started = [
-            run for run in job_runs if run['start_time'] is not None
+            run['start_time'] for run in job_runs if run['start_time'] is not None
```

Comment:
I think we should use `run['start_time'] or run.get('run_time')`.

With the current logic, if the run history looks like:
```
5 scheduled
4 failed with trigger timeout (no start time)
3 succeeded (has start time)
```
I think this script will return a realert_every interval based on the difference between 3 and 5, which is incorrect. The interval should be the same as the interval between each run.